cmd/cloud/dbt.go

```diff
@@ -0,0 +1,150 @@
+package cloud
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	cloud "github.com/astronomer/astro-cli/cloud/deploy"
+	"github.com/astronomer/astro-cli/cloud/deployment"
+	"github.com/astronomer/astro-cli/config"
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+)
+
+var (
+	mountPath      string
+	dbtProjectPath string
+
+	DeployBundle = cloud.DeployBundle
+)
+
+const (
+	dbtDeployExample = `
+Specify the ID of the Deployment on Astronomer you would like to deploy this dbt project to:
+
+  $ astro dbt deploy -d <deployment ID>
+
+Menu will be presented if you do not specify a deployment ID:
+
+  $ astro dbt deploy
+`
+
+	dbtDefaultMountPathPrefix = "/usr/local/airflow/dbt/"
+	dbtProjectYmlFilename     = "dbt_project.yml"
+	dbtBundleType             = "dbt"
+)
+
+func newDbtCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "dbt",
+		Short: "Manage your dbt projects deployed to Deployments running on Astronomer",
```

Comment:
```suggestion
		Short: "Deploy your dbt project to a Deployment on Astro",
```

Stays consistent with other deploy command and plus since Astronomer has multiple products managed by cli, gives clarity which product is this command for